src/sentry/models/groupsubscription.py

```diff
@@ -82,22 +82,32 @@ def subscribe_actor(
             else:
                 # subscribe the members of the team
                 team_users_ids = list(actor.member_set.values_list("user_id", flat=True))
-                return self.bulk_subscribe(group, team_users_ids, reason)
+                return self.bulk_subscribe(group=group, user_ids=team_users_ids, reason=reason)
 
         raise NotImplementedError("Unknown actor type: %r" % type(actor))
 
     def bulk_subscribe(
         self,
         group: Group,
-        user_ids: Iterable[int],
+        user_ids: Iterable[int] | None = None,
+        teams: Iterable[Team] | None = None,
         reason: int = GroupSubscriptionReason.unknown,
     ) -> bool:
         """
-        Subscribe a list of user ids to an issue, but only if the users are not explicitly
+        Subscribe a list of user ids and/or teams to an issue, but only if the users/teams are not explicitly
         unsubscribed.
         """
         # Unique the IDs.
-        user_ids = set(user_ids)
+        if user_ids:
+            user_ids = set(user_ids)
+        else:
+            user_ids = set()
```

Comment:
Python's one-liners are great for this sort of thing 

```suggestion
        user_ids = set(user_ids) if user_ids else set()
```